src/irclog2html/logs2html.py

```diff
@@ -74,7 +74,7 @@ def __ne__(self, other):
     def newfile(self):
         """Check whether the log file is new.
 
-        The log file is new iff the corresponding html file does not exist
+        The log file is new if the corresponding html file does not exist
```

Comment:
This might not be a typo, `iff` means "if and only if".

Reply:
Wasn't aware of this, though I think "if" and "if and only if" do mean the same in the end, otherwise using "if" wouldn't make sense in the first place.

Reply:
Yep, this was meant to be "if and only if".  Although, "if" in informal language tends to mean the same, and is less confusing.